Mathlib/NumberTheory/TsumDivsorsAntidiagonal.lean

```diff
@@ -126,4 +128,22 @@ lemma tsum_pow_div_one_sub_eq_tsum_sigma {r : 𝕜} (hr : ‖r‖ < 1) (k : ℕ)
   rw [← h00]
   exact tsum_congr₂ <| fun b c ↦ by simp [mul_comm c.val b.val, pow_mul]
 
+omit [NormSMulClass ℤ 𝕜] in
+lemma tendsto_zero_geometric_tsum_pnat {r : 𝕜} (hr : ‖r‖ < 1) :
```

Comment:
Using Tannery's theorem might seem like overkill here, since the sum is explicitly computable as 1 / (1 - r^m) - 1 which visibly tends to zero. But it works, so why not use it.

("It is crushing the nut with the triphammer, an absurd extravagance of energy, but the nut is very effectually crushed all the same." – Sherlock Holmes)